File: HM326Project/pinyougou_parent/pinyougou_interface/src/main/java/cn/itcast/core/service/order/SeckillOrderService.java
package cn.itcast.core.service.order;

import cn.itcast.core.pojo.seckill.SeckillOrder;
import com.github.pagehelper.PageInfo;


public interface SeckillOrderService {

    /** 根据商家id查询秒杀订单*/
    public PageInfo<SeckillOrder> search(Integer page,Integer rows,SeckillOrder seckillOrder);


    /** 运营商查询秒杀订单*/
    public PageInfo<SeckillOrder> searchForManager(Integer page,Integer rows,SeckillOrder seckillOrder);
}

File: HM326Project/pinyougou_parent/pinyougou_web_shop/src/main/webapp/js/service/seckillOrderService.js
app.service('seckillOrderService',function($http){

    //读取列表数据绑定到表单中
    this.findAll=function(){
        return $http.get('../order/findAll.do');
    }
    //分页
    this.findPage=function(page,rows){
        return $http.get('../order/findPage.do?page='+page+'&rows='+rows);
    }

    //秒杀订单查询
    this.search=function(page,rows,searchEntity){
        return $http.post('../order/search.do?page='+page+"&rows="+rows, searchEntity);
    }
})